src/module/account/settings/SettingFormLayout.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import { FormButtonToolbar, FormContainer } from '@/shared/components/form/FormElements';
+import { FormProvider, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Button } from '@/shared/components/Button';
+import * as z from 'zod';
+import { SettingFormGroup } from './SettingFormGroup';
+
+const userSchema = z.object({
```

Comment:
being extremely picky here, maybe just call this `schema` or `formSchema`, or even `registerFormSchema` if you want to be explicit.

Reply:
okkk